src/compiler/diagnosticMessages.json

```diff
@@ -2060,6 +2060,14 @@
         "category": "Error",
         "code": 5054
     },
+    "Cannot write file '{0}' which is one of the input files.": {
+        "category": "Error",
+        "code": 5055
+    },
+    "Cannot write file '{0}' since one or more input files would emit into it.": {

```

Comment:
Cannot write file '{0}' because it would be overwritten by multiple input files.